x/ccv/provider/types/msg.go

```diff
@@ -210,8 +210,16 @@ func (msg MsgSubmitConsumerDoubleVoting) ValidateBasic() error {
 		return fmt.Errorf("double voting evidence cannot be nil")
 	}
 
-	if msg.InfractionBlockHeader.Header == nil {
-		return fmt.Errorf("infraction header cannot be nil")
+	if msg.InfractionBlockHeader == nil {
```

Comment:
These missing validations were causing the handler to panic during E2E tests. This issue stemmed from the Hermes relayer, which used to let the `InfractionBlockHeader` field empty.